modules/nw-metallb-layer2.adoc

```diff
@@ -7,8 +7,12 @@
 [id="nw-metallb-layer2_{context}"]
 = MetalLB concepts for layer 2 mode
 
-In layer 2 mode, the `speaker` pod on one node announces the external IP address for a service to the host network.
-From a network perspective, the node appears to have multiple IP addresses assigned to a network interface.
+In layer 2 mode, the `speaker` pod on one node announces the external IP address for a service to the host network. 
+From a network perspective, the node appears to have multiple IP addresses assigned to a network interface. 
+[NOTE]
+====
+Since layer 2 mode relies on ARP and NDP, in order for MetalLB to work the client must be on the same subnet of the nodes announcing the service.
```

Comment:
Should it also not be a recommendation to use CIDR or addresses in IpAddressPool be in same subnet as the nodes for Layer 2 mode?

Reply:
That's a valid point.
How about:
"Additionally, the IP assigned to the service must be on the same subnet of the network used by the client to reach the service" . 

(This is because the node can still be on multiple networks)